tidb-configuration-file.md

```diff
@@ -62,6 +66,7 @@ The TiDB configuration file supports more options than command-line parameters.
 
 - Determines whether to enable the `utf8mb4` character check. When this feature is enabled, if the character set is `utf8` and the `mb4` characters are inserted in `utf8`, an error is returned.
 - Default value: `false`
+- Since v6.1.0, whether to enable the `utf8mb4` character check is determined by the TiDB configuration item `instance.tidb_check_mb4_value_in_utf8` or the system variable `tidb_check_mb4_value_in_utf8`. `check-mb4-value-in-utf8` still takes effect. But if `check-mb4-value-in-utf8` and `instance.tidb_check_mb4_value_in_utf8` are set at the same time, TiDB will apply the value of `instance.tidb_check_mb4_value_in_utf8`.
```

Comment:
```suggestion
- Since v6.1.0, whether to enable the `utf8mb4` character check is determined by the TiDB configuration item `instance.tidb_check_mb4_value_in_utf8` or the system variable `tidb_check_mb4_value_in_utf8`. `check-mb4-value-in-utf8` still takes effect. But if both `check-mb4-value-in-utf8` and `instance.tidb_check_mb4_value_in_utf8` are set, the latter takes effect.
```